<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnPrincipal</class>
 <widget class="QMainWindow" name="vtnPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>622</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>Ingrese solo números</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>510</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>GUARDAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnlimpiar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="placeholderText">
     <string>Ingrese el nombre del usuario</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbtiposervicio">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Seleccionar</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Transporte Rapido</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Logistico Personas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Logistico Mercancias</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mensajeria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Paqueteria Urbana</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Transporte Corporativo De Personal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Traslados Turísticos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Distribución De Carga</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtApellido">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="placeholderText">
     <string>Ingrese el apellido del usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtcedula">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ingrese solo números</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="placeholderText">
     <string>Cedula del usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtcostservicio">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="placeholderText">
     <string>0.00$</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbNombre">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbApellido">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Apellido:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCedula">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cedula:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbRuta">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ruta:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCostServicio">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Costo del servicio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTipoServicio">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tipo Servicio:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>338</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btninsertar">
       <property name="text">
        <string>Insertar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnbuscar">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnactualizar">
       <property name="text">
        <string>Actualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btneliminar">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="cbruta">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Norte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NorOeste</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SurEste</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Este</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oeste</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtcedula</tabstop>
  <tabstop>txtNombre</tabstop>
  <tabstop>txtApellido</tabstop>
  <tabstop>cbruta</tabstop>
  <tabstop>txtcostservicio</tabstop>
  <tabstop>cbtiposervicio</tabstop>
  <tabstop>btnGuardar</tabstop>
  <tabstop>btnlimpiar</tabstop>
  <tabstop>btninsertar</tabstop>
  <tabstop>btnbuscar</tabstop>
  <tabstop>btnactualizar</tabstop>
  <tabstop>btneliminar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
